constants/csv_parser.ts

```diff
@@ -149,12 +150,14 @@ const csvToQuestionArray = (fileName: string): Question[] => {
       idMap.set(id, index)
       const answers = answersMap.get(id) || []
       // determine the question type based on the answers
-      const type =
+      const type: QuestionType =
```

Comment:
Ternaries are always cool! But sometimes it's less readable when nested 🥴. Thoughts?

Reply:
So true, this is my bad actually